crates/iracing-data-api-client/docs/ConstantsApi.md

```
@@ -0,0 +1,92 @@
+[**models::IracingApiResponse**](iracingAPIResponse.md)
+
+### Authorization
+
+[bearerAuth](../README.md#bearerAuth)
+
+### HTTP request headers
+
+- **Content-Type**: Not defined
+- **Accept**: application/json
+
+[[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
```

markdownlint-cli2 (0.22.0)

[warning] No empty links

(MD042, no-empty-links)